### PR TITLE
Feat/met 5048 disable for maintenance

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

**Review by jeortizquan — CHANGES_REQUESTED** [review]
Tested locally and it works we need to take into account the caching time and the polling time so the maintenance message is displayed. just have an small observation about the field name.
afbeelding

File: projects/sandbox/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,84 +1,123 @@
...
+  ClickService,
+  MockModalConfirmService,
+  ModalConfirmComponent,
+  ModalConfirmService
+} from 'shared';
 
 describe('AppComponent', () => {
   let app: AppComponent;
   let fixture: ComponentFixture<AppComponent>;
+  const b4Each = (): void => {
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/app/app.component.spec.ts:24]: `beforeEach` is a command so can't be reused as a variable name - that's why I use [b4each](url), here and throughout the projects.

File: projects/metis/src/environments/maintenance-settings.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+} from '@europeana/metis-ui-maintenance-utils';
+
+const getEnvVar = (key: string): string | null => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const env = (window as any).__env;
+  return env ? env[key] : null;
+};
+
+export const maintenanceSettings: MaintenanceSettings = {
+  pollInterval: 60000,
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/metis/src/environments/maintenance-settings.ts:13]: Milliseconds are assumed by default in all the front-end projects.  In Metis-UI alone we have:

```
-   intervalStatusShort: 1000,
-   intervalStatus: 2500,
-   intervalStatusMedium: 5000,
-   intervalStatusLong: 60000,
-   intervalStatusMax: 60000 * 9.5,

``` 
In JS generally milliseconds are assumed -  `new Date().getTime()` exists, but `new Date().getTimeInMilliseconds()` doesn't.

File: projects/sandbox/src/environments/maintenance-settings.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+} from '@europeana/metis-ui-maintenance-utils';
+
+const getEnvVar = (key: string): string | null => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const env = (window as any).__env;
+  return env ? env[key] : null;
+};
+
+export const maintenanceSettings: MaintenanceSettings = {
+  pollInterval: 60000,
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/environments/maintenance-settings.ts:13]: Please see above.

File: projects/sandbox/src/app/app.component.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeortizquan) [reply projects/sandbox/src/app/app.component.spec.ts:24]: I see. thanks for the explanation.

File: projects/metis/src/environments/maintenance-settings.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeortizquan) [reply projects/metis/src/environments/maintenance-settings.ts:13]: my suggestion came from when reading the code is easier to remember what are the units of the variable.

**Pull request merged**